NAMESPACE

```diff
@@ -50,6 +50,7 @@ export(gt_output)
 export(gt_preview)
 export(gtsave)
 export(html)
+export(id_random)
```

Comment:
Thoughts on `id_random` vs `random_id`?

Reply:
The **highcharter** package has an exported function called `random_id()`. Do you think we should just keep using `random_id()` and basically not care about the conflict? `random_id()` is a way better name so I'd probably like to keep it.

Reply:
`gt_it()`?

Reply:
That solves the namespacing issue but doesn't give the impression that a random ID is being created. Really though, if we use `random_id()` from **highcharter** on accident, it would still create a random ID (just a bit shorter in length).